### PR TITLE
Update suggestions box

### PR DESCRIPTION
**Pull request by meel-hd**
### Fixed suggestions box
The actions box no longer covers the text field under it. Also fixed spacing between the buttons inside the box.
### Preview
image

Fixes: #8356

## Checklist

- [X] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [X] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

### CODE REVIEW

File: weblate/static/style-bootstrap.css [hunk 1]
```diff
@@ -602,20 +602,26 @@ form + .login-label {
...
+  border-radius: 5px;
   position: relative;
-  top: -5px;
+  transform: translateY(-100%);
   z-index: 10;
-  padding: 0 2px;
 }
 
-.btn-float .btn,
-.btn-padded .btn {
```
Comment (nijel) [inline weblate/static/style-bootstrap.css:616]: Why `btn-padded` is removed?

File: weblate/static/style-bootstrap.css [hunk 2]
```diff
@@ -1101,10 +1107,6 @@ thead th {
   border: 1px solid #2eccaa;
 }
 
-.btn-link {
```
Comment (nijel) [inline weblate/static/style-bootstrap.css:1104]: This class is used in many other locations, I'd expect this breaks rendering there.

File: weblate/static/style-bootstrap.css (hunk 1 above)
Comment (meel-hd) [reply weblate/static/style-bootstrap.css:616]: It applies smaller padding to the buttons than the default, it causes users to miss click. When removed the buttons reset to their default padding which looks better.

Comment (meel-hd) [reply weblate/static/style-bootstrap.css:616]: It applies smaller padding to the buttons than the default, it causes users to miss the click. When removed the buttons reset to their default padding which looks better.

File: weblate/static/style-bootstrap.css (hunk 2 above)
Comment (meel-hd) [reply weblate/static/style-bootstrap.css:1104]: Indeed. it is used on many elements. Also in the css there are multiple definitions of this class. I assume the default padding applied by bootstrap is enough. But I will leave the class styles here and remove it from the element instead.

Comment (nijel) [reply weblate/static/style-bootstrap.css:1104]: Removing the CSS makes these padding smaller, look at zen mode.

Before this PR:

obrazek

With this PR applied:

obrazek

**Comment by meel-hd** [comment]
# Preview
image

**Comment by nijel** [comment]
This needs more changes:

* The content of the suggestion is still not right aligned for RTL languages, but starts at the toolbar box:
  
  image
* This breaks comments layout which uses same buttons:
  
  obrazek

What about making it a toolbar for a full width of the comment/suggestion?

**Comment by meel-hd** [comment]
**Comments tooltip now look like this:**
image
**And RTL text is fixed:**
image

**Comment by meel-hd** [comment]
> What about making it a toolbar for a full width of the comment/suggestion?

I don't think it will look better. It will only consume more space than it needs.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**